package/MDAnalysis/wham.py

```diff
@@ -0,0 +1,644 @@
+        time step units. Only used for boostrap error; will be set to 1 for 
+        each window if not provided.
+    timeseries_data : str
+        Name of the auxiliary containing the force/reaction coordinate value 
+        throughout simulation.
+    energy : str, optional
+        Name of the auxiliary containing potential energy. [Currently must be
+        at exactly the same same steps as timeseries above (ie same dt and 
+        initial time)]. [I assume units must match energy_units?]. Only
+        required if simulations performed at different temperatures.
+
+    Returns
+    -------
+    data_names : dict
+        Dictionary containing the set if auxiliary/metadata names for *bundle*.
+    """
+    if not isinstance(bundle, dtr.Bundle):
+        TypeError('{} is not a bundle'.format(bundle))
+    check_bundle_metadata(bundle, [spring, loc_win_min, temperature])
+    if correl_time:
+        check_bundle_metadata(bundle, correl_time)
+    check_bundle_auxiliaries(bundle, [timeseries_data])
+    if energy:
+        check_bundle_auxiliaries(bundle, [energy])
+    data_names = locals()
+    data_names.pop('bundle')
+    return data_names
+
+
+def run_grossfield_wham(periodicity='', hist_min=None, 
+                        hist_max=None, num_bins=200, tol=1e-6, 
+                        calc_temperature=None, numpad=0, metafile=None, 
+                        outfile=None, num_MC_trials=200, check_only=False, 
+                        run_bootstrap=True):
+    """
+    Check the arguments for running WHAM w/ Grossfield implementation,
+    then run command.
+
+    [[ ARGS NEED SORTING ]]
+
+    Parameters
+    ----------
+    perodicity : str, optional
+        Periodicity of system. Default ('') indicates a nonperiodic reaction
+        coordiante.
+    num_bins : int, optional
+        Number of bins to use in histogram (= number of points in final profile).
+    tol : float, optional
+        Reference value to assess convergence (will stop iteration when the 
+        biggest change in F-value is less than this).
+    numpad : int, optional
+        ['padding' values, for periodic PMFs; for nonperiodic use 0 (default)]
+    num_MC_trials : int, optional
+        Number of 'fake' data sets to create, if running bootstrap error analysis
+    metafile : str
+        [[ filename for writting metadata file ]]
+    outfile : str
+        [[ filename for writting final output ]]
+    check_only : bool
+        [[ (Default False). If True, only check values are valid; exit before 
+         running wham command ]]
+    run_boostrap : bool, optional
+        Whether to run Monte Carlo bootstrap error analysis.
+    calc_temperature, hist_min, hist_max
+        [[ TBA?, see wham above ]]
+    """
+    wham_args = locals()
+    ## TODO - need to provide path to runfile? where best to do this? (check
+    ## it's actually wham...)
+    wham_command = '~/wham/wham/wham'
```

Comment:
This should be an argument. We need to find a strategy to find the binary if no path is provided.